src/main/resources/jenkins/plugins/office365connector/Webhook/help-timeout.html

```diff
@@ -1 +1 @@
-<div align="help">Sets connection timeout (in milliseconds) for TCP and HTTP. Default timeout is 30 seconds (30,000 ms)</div>
\ No newline at end of file
+<div align="help">Sets connection timeout (in milliseconds) for TCP and HTTP. Default timeout is 30 seconds (30,000 ms)</div>
```

Comment:
as the default value is already used https://github.com/jenkinsci/office-365-connector-plugin/pull/24/files#diff-bd70be8a13679d8af35645c64df15ceaR55 we don't have to mention it one more time

Reply:
I concur!